### ISSUE TITLE
Export the module name

### ISSUE THREAD

**Issue by kentcdodds** [angular-translate/angular-translate#944]
See [my post](https://medium.com/@kentcdodds/how-to-distribute-your-angularjs-module-e04d4dd58ddc) about it and [this angular.js PR comment](https://github.com/angular/angular.js/pull/10732#issuecomment-74986139) for reasons as to why it's a good idea. Basically, enables this:

```javascript
angular.module('myModule', [require('angular-translate')]);
```

**Comment by knalli** [comment]
Ehm.. yeah, nice addition. But only available in AJS 1.4 beta, and we are still supporting 1.2. ;)

**Comment by kentcdodds** [comment]
Actually, it's available as of 1.3.14. But no, I'm not saying that angular-translate needs to consume angular in that way, I'm just saying that angular-translate should, itself, export the module name using UMD.

**Comment by knalli** [comment]
Ok.. that was not clear by this issue. ;)

**Comment by kentcdodds** [comment]
Yeah, sorry :-)

**Comment by knalli** [comment]
Landed as 0a9cd78

**Comment by kentcdodds** [comment]
Sweet glory! :+1: Thanks!
